CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 3.0.0
```

Comment:
Would you merge all of the v3 changelog entries into one?

Reply:
Done.